_posts/august-2024-newsletter.md

```diff
@@ -0,0 +1,64 @@
+ 
+## DART Board Meeting
+
+---
+
+The first meeting is the DART Board meeting at [DART Headquarters](https://maps.app.goo.gl/36uCk3suvRdJPfM47)on August 27, 2024 at 6pm, please arrive 30 minutes early to register for public comments.
+
+Some members of the DART board are not very supportive of transit. Right now, there is a push among some board members to cut DART’s operating budget by 5-10% this year, and cap DART’s revenue growth. Other DART board members have expressed willingness to compromise with these cities that want to cut DART’s funding.
+
+We are here to to communicate 3 things
+
+1. More frequency/service improvements in next year's budget.
+2. Our opposition to these draconian budget cut resolutions, and our disappointment with the member cities who passed them
+3. Our opposition to ANY revenue capping.
+4. Express how DART should oppose any state legislation that cuts/caps their budget
+
+## Dallas City Council Meeting
+
+---
+The second meeting is the City of Dallas City Council meeting at [City Hall](https://maps.app.goo.gl/BgTdpbSjKRe73pF88) on August 28, 2024 at 9am. We intend to make a big showing here in support of DART funding, and communicate that we are:
+
+
+1. ***Opposed*** to DART Cuts
+2. ***Opposed*** to DART revenue caps
+3. ***Opposed*** to any state legislation that seeks to cap/cut DART’s revenue. We do NOT need to get the state legislature involved in DART’s budget.
+
+
+This meeting is just one day after the DART board meeting. If you can only attend one meeting, Dallas residents should prioritize this one.
+
+Registration for public comments will close at 5pm on August 27th.If you plan on attending this meeting in-person, please let us know so we can get a count. The City of Dallas also allows speaking virtually as an option in the event you are unable to attend due to other obligations.
+
+In general, registration will open on the 22nd at 8:15 am, you can register to speak 3 ways.
+1. Register online at [personalfinance.dallascityhall.com/speakers.aspx](https://personalfinance.dallascityhall.com/speakers.aspx)
+2. Email [secspeakerrequest@dallas.gov](mailto:secspeakerrequest@dallas.gov) with your name, address, phone number and meeting method (teleconference or in-person)
+3. Call the office at [214-670-3738](tel:+12146703738) with your name, address, phone number and meeting method (teleconference or in-person)
+
+ > As a reminder, the City of Dallas limits speakers to one public comment per month so please be aware of that. You can read the full policy on the [Dallas City Council website](https://dallascityhall.com/government/citysecretary/pages/ccrules.aspx). 
+
+The DART board meeting and the Dallas City Council Meeting we plan on attending are back to back, the board meeting is Tuesday Evening, the Dallas City Council meeting is Wednesday morning.
+
+Please make sure to follow all of our social media for the latest updates and news.
+
+Thanks,
```

Comment:
Either should have no comma or should have someone's name or "DATA", or whatever

Reply:
Changed to less of a letter